src/updateinfo.h

```diff
@@ -50,9 +50,20 @@ typedef struct {
     GStringChunk *chunk;
 } cr_UpdateCollectionPackage;
 
+typedef struct {
+    gchar *name;
+    gchar *stream;
+    gchar *version;
```

Comment:
@kontura,  is there a reason for a `version` to be of `gchar` type? 
In [libmodulemd](https://github.com/fedora-modularity/libmodulemd/blob/master/modulemd/v2/modulemd-module-stream.c#L31) it's `guint64`.

Reply:
I have updated it to be `guint64` as well. 
One thing I am not sure about, what should happen when the parsing of the `version` number fails? Is setting it to 0 ok?